.github/workflows/deploy.yml

```
@@ -0,0 +1,94 @@
+name: 자바 CD (Gradle & ECS)
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)